Updates to choose your teacher training course page merging some of the choosing course provider blog post (#3701)
DFE-Digital/get-into-teaching-app: app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training-course.md

* what you need to get onto the course
* the qualifications awarded by the course
* if the course is fee-funded or salaried 
* the type of course provider 
* if the course is full-time or part-time 

## What you need to get onto the course

Some training courses have specific eligibility requirements. 

It's important to check the requirements for each course you apply to on the training provider's course page. This is so you do not waste time on an application you’re unlikely to be successful with. 

### Your qualifications 

> You need to have a degree in any subject to be able to teach. But the degree grade that providers will accept may vary, for example it may be a 2:1 or a 2:2. So you’ll need to check each course provider’s requirements on their course page. [Find out about the qualifications you need to teach](/is-teaching-right-for-me/qualifications-you-need-to-teach).

Author: @gemmadallmandfe I initially took it out but when I read it back I didn't know if it sounded a bit odd without it. Or if all users would know what a degree grade was without an example.

> You need to have a degree in any subject to be able to teach. But the degree grade that providers will accept may vary, for example it may be a 2:1 or a 2:2. So you’ll need to check each course provider’s requirements on their course page. [Find out about the qualifications you need to teach](/is-teaching-right-for-me/qualifications-you-need-to-teach).

Reviewer: ```suggestion
You need to have a degree in any subject to be able to train to teach. But the degree grade that course providers will accept may vary. For example, most providers will accept a degree grade of 2:2 or higher, but some may require you to have a 2:1. So you’ll need to check each provider’s requirements on their course page. [Find out about the qualifications you need to teach](/is-teaching-right-for-me/qualifications-you-need-to-teach).
```
You need a degree to **train**; you need QTS in most schools to actually teach if that makes sense?

If we are going to add in degree grades, suggest we use the original text which gives a bit more context?

## If the course is fee-funded or salaried

### Fee-funded teacher training

Most teacher training courses are fee-funded. This means you have to pay tuition fees and will not earn a salary while you train. 

There are ways to fund your training, for example, through tuition fee and maintenance loans. You may also be eligible for a bursary or scholarship.

You can [find out about the different ways to fund your training](/funding-and-support).

<div class="inset">

<p> If you’re a non-UK citizen with settled status or indefinite leave to remain in the UK,  you may be eligible for a tuition fee loan, maintenance loan, bursary or scholarship. </p> 

<p> Other non-UK citizens will not be eligible for student finance, but may be eligible for a bursary or scholarship to train to teach languages and physics. <a href="/non-uk-teachers/fees-and-funding-for-non-uk-trainees#check-your-immigration-status-and-eligibility-for-financial-support">Check your immigration status and eligibility for financial support</a>.</p>

</div>

### Salaried teacher training

Some courses are salaried, which means you’ll receive an unqualified teacher’s salary while you train. 

> Places on salaried courses are limited and very competitive. If you’re a non-UK citizen, you're very unlikely to get a place on a salaried course. You can [find out what you’ll need to get on a salaried teacher training course](/funding-and-support/salaried-teacher-training). 

Reviewer: Is it worth putting the link on a separate line - is there any danger that UK citizens might not realise that link is for them too as it comes straight after the non-UK sentence?

(But I agree that moving the non-UK sentence up makes sense here)

Most teacher training courses will award QTS, but some will just award a [postgraduate certificate in education (PGCE)](/train-to-be-a-teacher/what-is-a-pgce), and some will award both. 

If your teacher training course leads to QTS, you may be eligible for a scholarship or bursary to help you train.

## If the course is fee-funded or salaried

### Fee-funded teacher training

Most teacher training courses are fee-funded. This means you have to pay tuition fees and will not earn a salary while you train. 

There are ways to fund your training, for example, through tuition fee and maintenance loans. You may also be eligible for a bursary or scholarship.

You can [find out about the different ways to fund your training](/funding-and-support).

<div class="inset">

<p> If you’re a non-UK citizen with settled status or indefinite leave to remain in the UK,  you may be eligible for a tuition fee loan, maintenance loan, bursary or scholarship. </p> 

> <p> Other non-UK citizens will not be eligible for student finance, but may be eligible for a bursary or scholarship to train to teach languages and physics. <a href="/non-uk-teachers/fees-and-funding-for-non-uk-trainees#check-your-immigration-status-and-eligibility-for-financial-support">Check your immigration status and eligibility for financial support</a>.</p>

Reviewer 2: [Learn more about fees and funding for non-UK trainees](/non-uk-teachers/fees-and-funding-for-non-uk-trainees)

Reviewer 2: (new link as per our conversation)

Most schools in England need you to have [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts) to teach. 

> Without QTS, you will not be eligible for a scholarship or bursary to help you train. You also may not be fully qualified to teach in your chosen school and will not receive the same pay and support when you start teaching.

Reviewer: I know this is in the original content but I wonder if it is slightly misleading on two counts:

* scholarships and bursaries are only available for certain subjects
* scholarships and bursaries are for courses leading to QTS - the way it is written implies you need QTS first?

Is there is a better way of wording this?

And once we've done that, it _may_ make more sense to return this section to the original order?

Most teacher training courses will award QTS, but some will just award a [postgraduate certificate in education (PGCE)](/train-to-be-a-teacher/what-is-a-pgce), and some will award both. 

## If the course is fee-funded or salaried

### Fee-funded teacher training

Most teacher training courses are fee-funded. This means you have to pay tuition fees and will not earn a salary while you train. 

There are ways to fund your training, for example, through tuition fee and maintenance loans. You may also be eligible for a bursary or scholarship.

You can [find out about the different ways to fund your training](/funding-and-support).

<div class="inset">

> <p> If you’re a non-UK citizen with settled status or indefinite leave to remain in the UK,  you may be eligible for a tuition fee loan, maintenance loan, bursary or scholarship. Other non-UK citizens will not be eligible for student finance, but may be eligible for a bursary or scholarship if they train to teach languages and physics. <a href="/non-uk-teachers/fees-and-funding-for-non-uk-trainees#check-your-immigration-status-and-eligibility-for-financial-support">Check your immigration status and eligibility for financial support</a>.</p>

Reviewer: ```suggestion
<p> If you’re a non-UK citizen with settled status or indefinite leave to remain in the UK,  you may be eligible for a tuition fee loan, maintenance loan, bursary or scholarship. Other non-UK citizens are not eligible for student finance, but you may be eligible for a bursary or scholarship if you train to teach languages and physics. <a href="/non-uk-teachers/fees-and-funding-for-non-uk-trainees#check-your-immigration-status-and-eligibility-for-financial-support">Check your immigration status and eligibility for financial support</a>.</p>
```
This feels like a lot of information - worth splitting into two paras?

Also it feels odd that we talk about `you` and `they`?

Finally GDS guidance is to avoid anchor links (https://www.gov.uk/guidance/content-design/links) - whilst we might need to keep this for now, this is something we should look to phase out as we do the integration work

Most schools in England need you to have [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts) to teach. 

Without QTS, you will not eligible for a scholarship or bursary to help you train. You also may not be fully qualified to teach in your chosen school and will not receive the same pay and support when you start teaching.

Most teacher training courses will award QTS, but some will just award a [postgraduate certificate in education (PGCE)](/train-to-be-a-teacher/what-is-a-pgce), and some will award both. 

## If the course is fee-funded or salaried

### Fee-funded teacher training

Most teacher training courses are fee-funded. This means you have to pay tuition fees and will not earn a salary while you train. 

> There are ways to fund your training, for example, through tuition fees and maintenance loans. You may also be eligible for a bursary or scholarship.

Reviewer: @MaxineCl `tuition fee` is correct - there is a tuition fee loan and a maintenance loan (see https://www.gov.uk/teacher-training-funding)

Author: @gemmadallmandfe changed it back!

* a school or group of schools (sometimes referred to as school-led training, or an apprenticeship) 
* [Teach First](https://www.teachfirst.org.uk/) (a charitable organisation)  

> Once you’ve found a course you’re interested in you can talk to the training provider before you apply. For example, you might want to find out: 

Reviewer: ```suggestion
Once you’ve found a course you’re interested in, you can talk to the training provider before you apply. For example, you might want to find out: 
```

Reviewer: I find this hard to understand without a comma splitting up - is it best practice not to have one?
